Trackovid19-web/src/manifest.webmanifest

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Covidografia",
+    "short_name": "Covidografia",
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
+    "display": "standalone",
+    "scope": "./",
+    "start_url": "./",
+    "icons": [
+        {
+            "src": "assets/android-chrome-192x192.png",
```

Comment:
This asset does not exist.

Reply:
It exist, but i forgot to update the path.
Good catch!